### ISSUE TITLE
Example of using S-BERT for Extractive Summarization

### ISSUE THREAD

**Issue by FadelMF** [UKPLab/sentence-transformers#2175]
I tried to fine-tune the S-BERT with my own summarization dataset, but I don't know what to do cause all examples to use labels (0-1) in there, and my dataset only has a gold summary and text article. Does anyone know the steps or examples so I can try them? Do I split the gold summary or text article into lists of sentences, or just long sentences into the pre-trained model?

I really appreciate any help you can provide.
